plane/fit_plane.py

```diff
@@ -5,71 +5,54 @@
 class FitPlane:
     
     """ Begin constractor methods """
-    def __init__(self,u=None,v=None,h=None,recommended_center_pix=[0,0]):
+    def __init__(self,u=None,v=None,h=None):
         self.u = np.array(u)
         self.v = np.array(v)
         self.h = np.array(h)
-        self.recommended_center_pix = np.array(recommended_center_pix)
-        
+
         if u is not None and v is not None and h is not None:
             self._check_u_v_consistency_assumptions()
     
     @classmethod
-    def from_fitting_points_on_photobleach_lines(cls, 
-        fluorescence_image_points_on_line_pix, photobleach_line_position_mm, photobleach_line_group,
-        print_inputs = False):
+    def from_template_centers(cls, template_center_positions_uv_pix, template_center_positions_xyz_um, print_inputs = False):
         """
-        This function initialize FitPlane by points on photobleach lines.
-        
+        This function initializes a FitPlane by a list of points for template centers.
+
         INPUTS:
-            fluorescence_image_points_on_line_pix: For each of the photobleach lines, 
-                find at least two points in the fluorescence image. Mark the coordinates as pixel values
-                l1 = [[x1,y1],[x2,y2],...] and create an array of those [l1,l2,...,ln]
-            photobleach_line_position_mm: an array defining the position (in mm) of each of the photobleach line positions 
-            photobleach_line_group: an array defining each line is a horizontal or vertical line ['h','v',...] etc
+            template_center_positions_uv_pix: For each photobleach barcode, find the center position in pixels. This is an
+                array of these center points.
+            template_center_positions_xyz_um: An array of points defining the position (in um) of the locations that each of 
```

Comment:
Can you give an example? Is it numpy.zeros(n,3) or numpy.zeros(3,n) where the "3" dimension is x,y,z and n is number of points?